src/templates/student/base.html

```diff
@@ -43,8 +43,14 @@
                      style="margin-left: auto; margin-bottom: 1px; margin-top: 1rem; margin-right: auto;">
                     <a class="thumbnail pull-left">
                         &nbsp &nbsp
-                        <img src="{% static '/img/cdclogo.png' %}" alt="IIT Jodhpur" class="logo"
+                        {% if user.is_authenticated %}
```

Comment:
Why are you putting this checks?

Reply:
@Tanuj22  to check if the user is logged in or not

Reply:
A student shouldn't be able to access this page if he is not logged in right?

Reply:
Oh, i misinterpreted it .I thought that if user is logged in then only the logo should redirect to home page.
I will change it.